SingularityService/src/main/java/com/hubspot/singularity/data/SingularityValidator.java

```diff
@@ -54,30 +54,29 @@
   private final int maxCpusPerRequest;
   private final int maxCpusPerInstance;
   private final int maxInstancesPerRequest;
-  private final int maxMemoryMbPerRequest;
-  private final int defaultCpus;
-  private final int defaultMemoryMb;
-  private final int defaultDiskMb;
   private final int defaultBounceExpirationMinutes;
+  private final int maxMemoryMbPerRequest;
   private final int maxMemoryMbPerInstance;
   private final boolean allowRequestsWithoutOwners;
   private final boolean createDeployIds;
   private final int deployIdLength;
   private final DeployHistoryHelper deployHistoryHelper;
   private final Resources defaultResources;
+  private final Pattern DAY_RANGE_REGEXP = Pattern.compile("[0-7]-[0-7]");
+  private final Pattern COMMA_DAYS_REGEXP = Pattern.compile("([0-7],)+([0-7])?");
```

Comment:
we should make these `private static final`, since they're constants that wont change between different instances of `SingularityValidator` (I know you could make the argument that since this is annotated with `@Singleton` it shouldn't matter, but it's still good code style to do this)